CONTRIBUTING.md

```diff
@@ -5,21 +5,22 @@ Thanks for your interest in contributing to `release-bot`.
 The following is a set of guidelines for contributing to `release-bot`.
 Use your best judgement, and feel free to propose changes to this document in a pull request.
 
-
 ## Reporting Bugs
-Before creating bug reports, please check a [list of known issues](https://github.com/user-cont/release-bot/issues) to see
-if the problem has already been reported (or fixed in a master branch).
+
+Before creating bug reports, please check a [list of known issues](https://github.com/user-cont/release-bot/issues) to see if the problem has already been reported (or fixed in a master branch).
 
 If you're unable to find an open issue addressing the problem, [open a new one](https://github.com/user-cont/release-bot/issues/new).
 Be sure to include a **descriptive title and a clear description**. Ideally, please provide:
- * version of release-bot you are using (`pip freeze | grep release-bot`)
- * the command you executed with output
+
+* version of release-bot you are using (`pip freeze | grep release-bot`)
+
+* the command you executed with output
```

Comment:
why the inserted blank lines?

Reply:
The linter was giving out warnings for new lines. :grimacing: